src/app/plugins/wolframalpha.plugin.ts

```diff
@@ -0,0 +1,78 @@
+import { MessageAttachment, MessageEmbed } from 'discord.js';
+import { Plugin } from '../../common/plugin';
+import { ChannelType, IContainer, IMessage } from '../../common/types';
+import Environment from '../../environment';
+// @ts-ignore
+import WolframAlphaAPI from 'wolfram-alpha-api';
+
+export class WolframAlphaPlugin extends Plugin {
+  private _defaultQuestion = 'What can you do?';
+  private _imageOptions = ['image', 'img'];
+  private _readableImageOption = this._imageOptions.join(' | ');
+  private _logoURL = 'https://www.symbols.com/images/symbol/2886_wolfram-alpha-logo.png';
+  private _errorMessage = "Sorry, I don't know that one";
+
+  public name: string = 'Wolfram Alpha';
+  public description: string =
+    'Ask wolfram alpha a question. \nProvide the first argument to get your answer as an image';
+  public usage: string = `wa <${this._readableImageOption}>? <question>`;
+  public pluginAlias = ['wa', 'wolfram', 'alpha', 'wolframalpha'];
+  public permission: ChannelType = ChannelType.Public;
+
+  constructor(public container: IContainer) {
+    super();
+  }
+
+  public async execute(message: IMessage, args: string[]) {
+    // connect to api
+    const waApi = WolframAlphaAPI(Environment.WolframAppID);
```

Comment:
we should be storing this in the constructor so that we do not spam connection open requests whenever the plugin is executed